content/advanced/330_servicemesh_using_appmesh/cleanup/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Cleanup"
+date: 2020-01-27T08:30:11-07:00
+weight: 100
+draft: false
+---
+
+#### Delete all the resources in the namespace
+
+```bash
+kubectl delete all --all -n prodcatalog-ns
+```
+
+#### Delete Product Catalog apps
+
+```bash
+kubectl delete namespace prodcatalog-ns
+```
+
+#### Delete the Product Catalog mesh
+
+```bash
+kubectl delete meshes prodcatalog-mesh
+```
+
+#### Uninstall the Helm Charts
+
+```bash
+helm -n appmesh-system delete appmesh-controller
+```
+
+#### Delete AWS App Mesh CRDs
+
+```bash
+for i in $(kubectl get crd | grep appmesh | cut -d" " -f1) ; do
+kubectl delete crd $i
+done
+```
+
+#### Delete the AWS App Mesh namespace
+
+```bash
+kubectl delete namespace appmesh-system
+```
+
+#### Delete the appmesh-controller service account
+```bash
+eksctl delete iamserviceaccount --cluster eksworkshop-eksctl   --namespace appmesh-system --name appmesh-controller
```

Comment:
Ideally we'll be using IRSA, but if you stick with the new MNG (or add policies to the existing MNG node instance role), we should remove the additional policies here.

Reply:
updated